sdks/v3-sdk/src/utils/computePoolAddress.ts

```diff
@@ -19,20 +21,24 @@ export function computePoolAddress({
   tokenB,
   fee,
   initCodeHashManualOverride,
+  chainId,
 }: {
   factoryAddress: string
   tokenA: Token
   tokenB: Token
   fee: FeeAmount
   initCodeHashManualOverride?: string
+  chainId?: ChainId
 }): string {
   const [token0, token1] = tokenA.sortsBefore(tokenB) ? [tokenA, tokenB] : [tokenB, tokenA] // does safety checks
-  return getCreate2Address(
-    factoryAddress,
-    keccak256(
-      ['bytes'],
-      [defaultAbiCoder.encode(['address', 'address', 'uint24'], [token0.address, token1.address, fee])]
-    ),
-    initCodeHashManualOverride ?? POOL_INIT_CODE_HASH
-  )
+  const salt = keccak256(['bytes'], [defaultAbiCoder.encode(['address', 'address', 'uint24'], [token0.address, token1.address, fee])])
+  const initCodeHash = initCodeHashManualOverride ?? POOL_INIT_CODE_HASH
```

Comment:
The initcode hash for zksync is 
`0x010013f177ea1fcbc4520f9a3ca7cd2d1d77959e05aa66484027cb38e712aeed`
whereas for all other chains its
`0xe34f199b19b2b4f47f68442619d555527d244f78a3297ea89325f843f87b8b54`

Reply:
thanks. I was thinking to override init code hash via the method parameter `initCodeHashManualOverride`, but it's prob better to take care in v3-sdk, since FE also uses v3-sdk to compose pool addresses